File: 8-4-17/JavaScriptOOP/JavaScriptOOP/Person.js
var Person = function (firstName, middleName, lastName) {
    this.FirstName = firstName;
    this.MiddleName = middleName;
    this.LastName = lastName;

    this.GetFullName = function () {
        return this.FirstName + "" + this.MiddleName + "" + this.LastName;
    }


}

File: 8-4-17/JavaScriptOOP/JavaScriptOOP/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace JavaScriptOOP
{
    public class Person
    {
        public string FirstName { get; set; }
        public string MiddleName { get; set; }

        public string LastName { get; set; }

        public string GetFullName()
        {
            return FirstName + "" + MiddleName + "" + LastName;
        }

        public Person()
        {
            
        }

        public Person(string fisrtName, string middleNme, string lastName)
        {
            FirstName = fisrtName;
            MiddleName = middleNme;
            LastName = lastName;

        }
    }
}